projects/boringssl/build.sh

```diff
@@ -48,3 +46,20 @@ for F in $fuzzerFiles; do
     zip -j $OUT/${fuzzerName}_seed_corpus.zip $SRC/boringssl/fuzz/${fuzzerName}_corpus/*
   fi
 done
+
+rm -rf genfiles && mkdir genfiles && $SRC/LPM/external.protobuf/bin/protoc asn1_pdu.proto --cpp_out=genfiles --proto_path=$SRC/fuzzing/proto/asn1-pdu-proto/
```

Comment:
Why this directory structure for `--proto-path`?

Reply:
So proto-path specifies where the proto is, and the proto lives in this github [repo](https://github.com/dannyallover/fuzzing/tree/master/proto/asn1-pdu-proto) which we clone in the Dockerfile

Not sure if there's a better way to do this?